airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1380,7 +1381,96 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/VersionInfo'
-
+  /plugins:
+    get:
+      summary: Get a list of loaded plugins
+      x-openapi-router-controller: airflow.api_connexion.endpoints.plugin_endpoint
+      operationId: get_plugins
+      tags: [Plugin]
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  number:
+                    type: string
+                    description: The plugin number
+                  name:
+                    type: string
+                    description: The name of the plugin
+                  hooks:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The plugin hooks
+                  executors:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The plugin executors
+                  macros:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The plugin macros
+                  admin_views:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The plugins admin views
+                  flask_blueprints:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The flask blueprints
+                  menu_links:
+                    type: array
+                    items:
+                      type: string
+                      nullable: true
+                    description: The menu links
```

Comment:
Should we remove `menu_links` and `admin_views` -- they are for the old UI

https://github.com/apache/airflow/blob/25fa3092323e0cd13b11bb8e809b88ff3c043e4e/airflow/plugins_manager.py#L316-L328

Reply:
Or atleast add "deprecated" in the API docs

Reply:
I removed it. I saw that they wasn't in plugins command and since this is new endpoint, letting it come without them looks good to me?